policy/policy.go

```diff
@@ -207,4 +228,5 @@ const (
 	tenDays       time.Duration = time.Duration(time.Hour * 24 * 10)
 	thirtyDays    time.Duration = time.Duration(time.Hour * 24 * 30)
 	fortyFiveDays time.Duration = time.Duration(time.Hour * 24 * 45)
+	oneYear       time.Duration = time.Duration(time.Hour * 24 * 365)
```

Comment:
Is there any reason to be concerned about leap years having 366 days?

Reply:
Yes, but check out the tests!

tldr: the algorithm for finding the following quarter accounts for this.